pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -321,3 +341,61 @@ def _setup_lookup(self, timestamps: np.ndarray) -> np.recarray:
         lookup.timestamp = timestamps
         lookup.container_idx = -1  # virtual container by default
         return lookup
+
+
+class RenameSet:
+    class RenameFile:
+        def __init__(self, path):
+            self.path = path
+
+        @property
+        def is_exists(self):
```

Comment:
Naming suggestion: `is_exists -> exists`. This would mirror `os.path.exists` on which it is based.